app/src/main/java/org/oppia/app/settings/profile/ProfileEditActivity.kt

```diff
@@ -10,7 +10,8 @@ const val KEY_PROFILE_EDIT_PROFILE_ID = "KEY_PROFILE_EDIT_PROFILE_ID"
 
 /** Activity that allows user to edit a profile. */
 class ProfileEditActivity : InjectableAppCompatActivity() {
-  @Inject lateinit var profileEditActivityPresenter: ProfileEditActivityPresenter
+  @Inject
```

Comment:
Bring this to single line.

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting